src/odemis/gui/win/acquisition.py

```diff
@@ -1443,3 +1443,23 @@ def ShowChamberFileDialog(parent, projectname):
     path = dialog.GetDirectory()
     fn = dialog.GetFilename()
     return os.path.join(path, fn)
+
+def LoadProjectFileDialog(parent, projectname):
```

Comment:
Maybe you can use this https://github.com/delmic/odemis/blob/77a66420671011a1e0e0d1a88aab64d1fe93919c/src/odemis/gui/comp/file.py#L41